README.md

```diff
@@ -42,6 +42,46 @@ If you would like to add to the list, please see our [contribution guidelines](.
   - [🇬🇧 UK](#-uk)
   - [🇻🇳 Vietnam](#-vietnam)
 
+- [Navigation](#navigation)
+- [List of Slack Channels for local communities by region](#list-of-slack-channels-for-local-communities-by-region)
+  - [🇺🇸 USA:](#-usa)
+    - [New England](#new-england)
+    - [Mid-Atlantic](#mid-atlantic)
+    - [The Midwest](#the-midwest)
+    - [The Southeast](#the-southeast)
+    - [The Southwest](#the-southwest)
+    - [The West](#the-west)
+- [Other Countries](#other-countries)
+    - [🇦🇷 Argentina](#-argentina)
+    - [🇦🇺 Australia](#-australia)
+    - [🇧🇷 Brazil](#-brazil)
+    - [🇨🇮 Cote d'Ivoire (Ivory Coast)](#-cote-divoire-ivory-coast)
+    - [🇨🇦 Canada](#-canada)
+    - [🇨🇱 Chile](#-chile)
+    - [🇩🇰 Denmark](#-denmark)
+    - [🇫🇮 Finland](#-finland)
+    - [🇫🇷 France](#-france)
+    - [🇩🇪 Germany](#-germany)
+    - [🇬🇭 Ghana](#-ghana)
+    - [🇮🇳 India](#-india)
+    - [🇮🇩 Indonesia](#-indonesia)
+    - [🇮🇪 Ireland](#-ireland)
+    - [🇮🇱 Israel](#-israel)
+    - [🇮🇹 Italy](#-italy)
+    - [🇯🇵 Japan](#-japan)
+    - [🇲🇾 Malaysia](#-malaysia)
+    - [🇲🇽 Mexico](#-mexico)
+    - [🇳🇿 New Zealand](#-new-zealand)
+    - [🇳🇬 Nigeria](#-nigeria)
+    - [🇵🇭 Philippines](#-philippines)
+    - [🇵🇱 Poland](#-poland)
+    - [🇷🇴 Romania](#-romania)
+    - [🇿🇦 South Africa](#-south-africa)
+    - [🇷🇸 Serbia](#-serbia)
+    - [🇬🇧 UK](#-uk)
+    - [🇺🇾 Uruguay](#-uruguay)
+    - [🇺🇦 Ukraine](#-ukraine)
+    - [🇻🇳 Vietnam](#-vietnam)
```

Comment:
You added a lot of extra duplicate categories. 
Please remove lines 45-84

Reply:
Oh, Didn't realize that, Sorry